core/qbft/qbft.go

```diff
@@ -39,7 +39,7 @@ type Value[V any] interface {
 type Transport[I any, V Value[V]] struct {
 	// Broadcast sends a message with the provided fields to all other
 	// processes in the system (including this process).
-	Broadcast func(typ MsgType, instance I, source int64, round int64, value V, pr int64, pv V, justify []Msg[I, V])
+	Broadcast func(typ MsgType, instance I, source int64, round int64, value V, pr int64, pv V, justification []Msg[I, V])
```

Comment:
Maybe
```suggestion
	Broadcast func(typ MsgType, instance I, source int64, round int64, value V, pr int64, pv V, justifications []Msg[I, V])
```

Reply:
technically it is a single justification containing multiple messages...